test/parallel/test-pipe-address.js

```diff
@@ -3,19 +3,13 @@ var common = require('../common');
 var assert = require('assert');
 var net = require('net');
 
-var address = null;
-
 var server = net.createServer(function() {
   assert(false); // should not be called
 });
 
 common.refreshTmpDir();
 
 server.listen(common.PIPE, function() {
```

Comment:
Wrapping the connection handler isn't always needed, depending on how the code is structured. In this case, it *is* needed, and I missed it. I'll fix it.